css/style.css

```diff
@@ -0,0 +1,131 @@
+
+* {
+  box-sizing: border-box;
+  padding:0px ;
+  margin: 0px;
+}
+body
+{
+  display: flex;
+  flex-wrap: wrap;
+  background-color:lightgreen;
+  font-family: Helvetica, sans-serif;
+}
+#head
+{
+  z-index: 1;
+  width: 100%;
+  height: 300px;
+  background-color: skyblue ;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  font-family: 'Acme', sans-serif;
+}
+#head h1{
+    font-size: 20vw;
+}
+#foot
+{
+  background-color: inherit;
+  border: 5px dotted black;
+  width:100%;
+  height:200px;
+  margin-bottom: 5px; 
+  z-index:1;
+}
+.timeline {
+  position: relative;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+.timeline::after {
+  content: '';
+  position: absolute;
+  width: 6px;
+  background-color: blue;
+  top: 0;
+  bottom: 0;
+  left: 50%;
+  margin-left: -3px;
+}
+.container {
+  padding: 10px 40px;
+  position: relative;
+  background-color: inherit;
+  width: 50%;
+}
+.container::after {
+  content: '';
+  position: absolute;
+  width: 25px;
+  height: 25px;
+  right: -17px;
+  background-color: white;
+  border: 4px solid green;
+  top: 15px;
+  border-radius: 50%;
+  z-index: 1;
+}
+.left {
+  left: 0;
+}
+.right {
+  left: 50%;
+}
+.left::before {
+  content: " ";
+  height: 0;
+  position: absolute;
+  top: 22px;
+  width: 0;
+  z-index: 1;
+  right: 30px;
+  border: medium solid white;
+  border-width: 10px 0 10px 10px;
+  border-color: transparent transparent transparent white;
+}
+.right::before {
+  content: " ";
+  height: 0;
+  position: absolute;
+  top: 22px;
+  width: 0;
+  z-index: 1;
+  left: 30px;
+  border: medium solid white;
+  border-width: 10px 10px 10px 0;
+  border-color: transparent white transparent transparent;
+}
+.right::after {
+  left: -16px;
+}
+.content {
+  padding: 20px 30px;
+  background-color: white;
+  position: relative;
+  border-radius: 6px;
+}
+@media screen and (max-width: 600px) {
+  .timeline::after {
+  left: 31px;
+  }
+ .container {
+  width: 100%;
+  padding-left: 70px;
+  padding-right: 25px;
+  }
+  .container::before {
+  left: 60px;
+  border: medium solid white;
+  border-width: 10px 10px 10px 0;
+  border-color: transparent white transparent transparent;
+  }
+  .left::after, .right::after {
+  left: 15px;
+  }
+ .right {
+  left: 0%;
+  }
+}
```

Comment:
You are not using this file. You may delete it I guess.

Reply:
not using the whole file anyways

Reply:
then delete the file and do push it. it will automatically reflect on this pull request